app/views/exam_templates/_form.html.erb

```diff
@@ -3,6 +3,12 @@
     <%= f.error_messages%>
     <div class='template-pane'>
       <h4>
+        <%= link_to t('exam_templates.delete.all_general'), action: 'destroy_all' %>
+      </h4>
+      <h4>
+        <%= link_to t('exam_templates.delete.general'), action: 'destroy', id: exam_template.id %>
```

Comment:
I think the link is about deleting associated objects, which might be useful to delete template divisions. But for deleting exam template objects, aren't we supposed to write our own method in the controller?